clients/src/main/java/org/apache/kafka/common/requests/DescribeConfigsResponse.java

```diff
@@ -174,19 +97,25 @@ public boolean isReadOnly() {
     }
 
     public enum ConfigSource {
-        UNKNOWN_CONFIG((byte) 0),
-        TOPIC_CONFIG((byte) 1),
-        DYNAMIC_BROKER_CONFIG((byte) 2),
-        DYNAMIC_DEFAULT_BROKER_CONFIG((byte) 3),
-        STATIC_BROKER_CONFIG((byte) 4),
-        DEFAULT_CONFIG((byte) 5),
-        DYNAMIC_BROKER_LOGGER_CONFIG((byte) 6);
+        UNKNOWN_CONFIG((byte) 0, org.apache.kafka.clients.admin.ConfigEntry.ConfigSource.UNKNOWN),
```

Comment:
It's unfortunate the public enum cannot do the mapping and we have to keep a copy here. Could we maybe rename this one so this looks less crazy? WDYT?